client/src/views/files/FoldersPage.vue

```diff
@@ -148,17 +148,19 @@
           v-if="children.length && displayView === DisplayState.Grid"
           class="folders-container-grid"
         >
-          <ion-item
+          <div
             class="folder-grid-item"
             v-for="child in children"
             :key="child.id"
           >
             <file-card
               :file="child"
+              :show-checkbox="selectedFilesCount > 0"
               @click="onFileClick"
               @menu-click="openFileContextMenu"
+              ref="fileItemRefs"
```

Comment:
This doesn't work, it's going to reference only one element. For `file-list-item` it works because the `v-for` is directly on the element, here the `v-for` is on the parent div. This is probably the reason why the selection is not working for files in grid display.

Reply:
I update this with a `file-list-item-refs` and `file-grid-item-refs`.